### ISSUE TITLE
block-indentation should ignore <pre>

### ISSUE THREAD

**Issue by mike-north** [ember-template-lint/ember-template-lint#702]
the `block-indentation` rule, in its current state, can be harmful around HTML tags where whitespace is meaningful (i.e., `<pre>`)

To reproduce
```html
      <pre
        class="bg-grey-lighter border border-grey-light text-grey-darkest text-sm font-mono rounded p-3 mt-2 overflow-scroll">.marquee-lightspeed { -webkit-marquee-speed: fast; }
.marquee-lightspeeder { -webkit-marquee-speed: faster; }</pre>
```
I get this linter feedback
```
15:62  error  Incorrect indentation for `pre` beginning at L14:C6.
Expected `</pre>` ending at L15:C62 to be at an indentation of 6 but was found at 56.
block-indentation
```
The whitespace I'd have to add to make the linter happy in this case would result in an undesirable visual change to the rendered HTML (i.e., an extra blank newline at the bottom of each `<pre>`)

### Suggested solution
ignore `<pre>` for this rule

---
##### Versions
ember-cli-template-lint: 1.0.0-beta.3
ember-template-lint: 1.1.0

**Comment by rwjblue** [comment]
Seems good to me, any other elements off the top of your head for this?

**Comment by mike-north** [comment]
> Seems good to me, any other elements off the top of your head for this?

I tried all the whitespace modes I could think of, and only found troubles around `<pre>` (i.e., `<code>` is fine as-is.

**Comment by rwjblue** [comment]
Somewhat mitigated by #899 (block-indentation is removed from the recommended set), but this should still be addressed.
